README.md

```diff
@@ -69,13 +69,13 @@ within the cloned directory:
 ```bash
 git clone git@github.com:trixi-framework/Trixi.jl.git
 cd Trixi.jl
-julia --project=. -e 'import Pkg; Pkg.instantiate()' # Install Trixi's dependencies
+julia --project=@. -e 'import Pkg; Pkg.instantiate()' # Install Trixi's dependencies
 julia -e 'import Pkg; Pkg.add("Trixi2Vtk"); Pkg.add("Trixi2Img")' # Install postprocessing tools
 ```
 If you installed Trixi this way, you always have to start Julia with the `--project`
 flag set to your local Trixi clone, e.g.,
 ```bash
-julia --project=.
+julia --project=@.
```

Comment:
If you look at `julia --help`, they document `@.` as argument for the current folder. `--project=.` can fail for paths containing non-ASCII characters (https://stackoverflow.com/questions/53613663/what-is-in-julia-project-command-line-option). I'm not sure how it's handled on Windows but it's usually better to let Julia handle paths instead of the underlying shell, I guess.

Reply:
OK